packages/web-app-files/src/services/archiver.ts

```diff
@@ -17,6 +17,8 @@ export interface ArchiverCapability {
   formats: string[]
   // eslint-disable-next-line camelcase
   archiver_url: string
+  max_num_files: string
```

Comment:
Why are those strings ?

Reply:
I'm just using what oCIS/reva provide on my recently pulled docker image ;) see https://github.com/cs3org/reva/blob/2be8a61879b22823c852757497c81f2d544f6f9d/internal/http/services/owncloud/ocs/data/capabilities.go#L114

Reply:
Couldn't we parse them beforehand?

Reply:
No, this only reflects the raw capabilities coming from the backend. Parsing makes sense in the capability composables though.